plugin/src/main/java/org/opensearch/sql/plugin/transport/TransportPPLQueryResponse.java

```diff
@@ -25,4 +31,22 @@ public TransportPPLQueryResponse(StreamInput in) throws IOException {
   public void writeTo(StreamOutput out) throws IOException {
     out.writeString(result);
   }
+
+  public static TransportPPLQueryResponse fromActionResponse(ActionResponse actionResponse) {
```

Comment:
Could you add UT to show the use case of this method? In which case, we want to convert other ActionResponse to PPL response?

Reply:
Added UT for this.